bin/elm-test

```diff
@@ -6,12 +6,30 @@ process.title = 'elm-test';
 
 var compile    = require("node-elm-compiler").compile,
   suffix       = require("../elm-io-ports.js"),
-  fs           = require("fs"),
+  fs           = require("fs-extra"),
+  path         = require("path"),
   temp         = require("temp").track(), // Automatically cleans up temp files.
   util         = require("util"),
   _            = require("lodash"),
   childProcess = require("child_process");
 
+if (process.argv[2] == "init") {
+  var copyTemplate = function(templateName) {
+    if (fs.existsSync(templateName)) {
+      console.log(templateName + " already exists");
+    } else {
+      fs.copySync(path.resolve(__dirname, "../templates/" + templateName), templateName);
+      console.log("Created " + templateName);
+    }
+  };
+
+  childProcess.execSync("elm package install deadfoxygrandpa/Elm-Test", {stdio:[0,1,2]});
+  childProcess.execSync("elm package install maxsnew/IO", {stdio:[0,1,2]});
```

Comment:
I'm hesitant about that since people using `elm-test` for the first time might want to get the prompt for the new packages.  But how about if we add a `--yes` option to `elm-test init` ?